buildenv/jenkins/common/build

```diff
@@ -167,10 +167,10 @@ def archive() {
             // https://github.com/eclipse/openj9/issues/1114
             // ghprbPullId is the PullRequest ID which only shows up in Pull Requests
             if (!params.ghprbPullId) {
-                sh "tar -zcvf ${WORKSPACE}/${TEST_PREFIX}`git -C openj9 rev-parse --short HEAD`${TEST_SUFFIX} openj9/test/"
```

Comment:
Please explain why `${WORKSPACE}` is removed. common/test still uses it, does it need to be removed there as well?

Reply:
By default the stage runs in the build workspace. Is not necessary to specify it.
On Windows itar fails due to the fact that `<WORKSPACE>` is not converted to cygpath it's a windows path C:\Users\...
In common/test is not necessary to remove it as java on cygwin uses windows paths not cygpaths.